lib_comfyui/webui/tab.py

```diff
@@ -58,6 +62,10 @@ def automatic_install_comfyui(install_location):
     install_comfyui.main(install_location)
     shared.opts.comfyui_install_location = install_location
 
+    if should_install_manager:
+        manager_install_location = os.path.join(install_location, 'custom_nodes', 'ComfyUI-Manager')
```

Comment:
consider making a function: comfyui path -> comfyui manager path, as we are duplicating this info in 2-3 places otherwise.

Reply:
also whenever possible prefer to use pathlib over os.path